tests/DocumentationExamplesTest.php

```diff
@@ -1180,6 +1184,223 @@ public function testIndex_example_2()
         $this->assertEquals('cuisine_1_name_1', $indexName);
     }
 
+    // Start Transactions Intro Example 1
+    private function updateEmployeeInfo1(Manager $manager, Session $session)
```

Comment:
AFAIK, none of the other examples refer to class names. If we do, I think we should use the FQCN. It's quite verbose, but it ensures the examples can be easily copy-pasted. Users can always add `use` statements on their own.

I find it odd that Session gets passed in here instead of creating it within, but I suppose that's up to the docs team.

Reply:
Done